src/Command/ExtractCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Translator\Extractor\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+use Yiisoft\Translator\Extractor\Extractor;
+use Yiisoft\Yii\Console\ExitCode;
+
+final class ExtractCommand extends Command
+{
+    /** @var string|null */
+    protected static $defaultName;
+
+    private string $defaultCategory = 'app';
+
+    private Extractor $extractor;
+
+    public function __construct(Extractor $extractor) {
+        self::$defaultName = 'translator/extract';
+        $this->extractor = $extractor;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Extracts translator IDs from files.')
+            ->addOption('languages', 'L', InputOption::VALUE_REQUIRED, 'Creating translations for languages. By default only `en`.', 'en')
+            ->addOption('category', 'C', InputOption::VALUE_REQUIRED, 'Default category for unsetted category messages.', $this->defaultCategory)
```

Comment:
Is it really required? I see it being set to `app` already.

Reply:
User may be wants extract message from another module, with changed translation category (for example with use  method `withCategory`)

Reply:
Yes but why making the option required?

Reply:
It'w wrong - fixed. But all Option in symfony/console optional